### PR TITLE
Swap order of processing json_request to prevent bad jsonlines request.

### PR DESCRIPTION
**Pull request by matherit**
*Issue #, if available:*

*Description of changes:*
Swaps order on how json requests are processed. This should prevent valid TFS requests from being processed as if its a jsonlines type request.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/build_artifacts/sagemaker/tensorflowServing.js [hunk 1]
```diff
@@ -140,7 +140,9 @@ function parse_custom_attributes(r) {
 function json_request(r) {
     var data = r.requestBody
 
-    if (is_json_lines(data)) {
+    if (is_tfs_json(data)) {
+        tfs_json_request(r, data)
+    } else if (is_json_lines(data)) {
         json_lines_request(r, data)
     } else if (is_tfs_json(data)) {
```
Comment (satishpasumarthi) [inline docker/build_artifacts/sagemaker/tensorflowServing.js:147]: this condition is redundant.

**Review by satishpasumarthi — CHANGES_REQUESTED** [review]
left a comment

**Review by satishpasumarthi — APPROVED** [review]
LGTM

**Review by mseth10 — APPROVED** [review]
LGTM

**Pull request merged**